examples.md

```diff
@@ -7,12 +7,11 @@ nav_order: 3
 
 # Example Projects
 
-## Tupelo Explorer and Wallet
+## [Tupelo Explorer and Wallet](/examples/decentracar)
```

Comment:
Why is this being linked to the Decentracar example?

Reply:
Fixorated.